student/models/enrolment.py

```diff
@@ -14,6 +14,9 @@
 class Enrolment(SafeDeleteModel):
     _safedelete_policy = SOFT_DELETE_CASCADE
 
+    COMPLETED = 'COMPLETED'
```

Comment:
Could these names be confusing? I believe by completed and not completed we are referring to graduated and not yet graduated, but in the context of enrolment someone might think that these are referring to successful and unsuccessful enrolment into the course?